src/ComponentTester.cpp

```diff
@@ -27,17 +27,19 @@ void ComponentTester::testPin(){
   }
   if(componentType.equalsIgnoreCase("PHOTORESISTOR")){
     for(int i = 1; i <= 10; i++){
-        /*Serial.print("test#");
-        Serial.print(i);
-        Serial.print(" ");
-        Serial.println(analogRead(pin));*/
+        if(Properties::DEBUGGING_ACTIVE) {
+          Serial.print("test# " + i + ": ");
```

Comment:
Ah - ok. Good catch. I didn't have my environment all configured on my work laptop so I didn't try to compile. I'll fix this.